src/backend/PortabilityService.WorkflowManagement/WorkflowActions/BaseAction.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace PortabilityService.WorkflowManagement
+{
+    abstract class BaseAction
```

Comment:
Nitpicking: an access modifier?

Reply:
Fixed: `internal` added for base and derived actions.

---
In reply to: [186502322](https://github.com/Microsoft/dotnet-apiport/pull/635#discussion_r186502322) [](ancestors = 186502322)